lib/core/src/lib/utils/object-utils.ts

```diff
@@ -65,4 +65,59 @@ export class ObjectUtils {
 
         return result;
     }
+
+    static isObject(target: any): boolean {
+        return target === Object(target);
+    }
+
+    static isEmpty(target: any): boolean {
+        return target && Object.keys(target).length === 0 && Object.getPrototypeOf(target) === Object.prototype;
+    }
+
+    static hasKeys(target: any): boolean {
+        return target && Object.keys(target).length > 0;
+    }
+
+    static isBooleanObject(target: any): boolean {
+        return Object.values(target).every(value => typeof value === "boolean");
```

Comment:
can you replace double quotes with single quotes?

Reply:
Sure!

Reply:
We should automate this kind of thing with ESLint though, @popovicsandras.

Formatting, line ending, whitespace trimming, and proper quotation... that should come out for free in VSCode, when leaving the line